pkg/tecdsa/dkg/member.go

```diff
@@ -333,3 +339,35 @@ func (sm *signingMember) initializeSubmittingMember() *submittingMember {
 type submittingMember struct {
 	*signingMember
 }
+
+// identityConverter implements the common.IdentityConverter for tECDSA DKG.
+// It maps every member index to a party ID by adding a constant seed value.
+type identityConverter struct {
+	seed *big.Int
+}
+
+func (ic *identityConverter) MemberIndexToTssPartyID(
+	memberIndex group.MemberIndex,
+) *tss.PartyID {
+	partyIDKey := ic.MemberIndexToTssPartyIDKey(memberIndex)
+
+	return tss.NewPartyID(
+		partyIDKey.Text(10),
+		fmt.Sprintf("member-%v", memberIndex),
+		partyIDKey,
+	)
+}
+
+func (ic *identityConverter) MemberIndexToTssPartyIDKey(
+	memberIndex group.MemberIndex,
+) *big.Int {
+	return new(big.Int).Add(ic.seed, big.NewInt(int64(memberIndex)))
+}
+
+func (ic *identityConverter) TssPartyIDToMemberIndex(
+	partyID *tss.PartyID,
+) group.MemberIndex {
+	return group.MemberIndex(
+		new(big.Int).Sub(partyID.KeyInt(), ic.seed).Int64(),
```

Comment:
We allow it to go <0 because we'll fail in `AggregateTssMessages` on `cannot get symmetric key with member ...` and no harm can be done this way, correct?

Reply:
Actually, I think we should return `MemberIndex(0)` in this case and make it clear in the docs.

Reply:
Will be done in https://github.com/keep-network/keep-core/pull/3262.

Reply:
Yep, this is one reason. Another one is the fact that should never happen if conversions are done through the converter.